pages/common/xargs.md

```diff
@@ -2,24 +2,33 @@
 
 > Execute a command with piped arguments coming from another command, a file, etc.
 > The input is treated as a single block of text and split into separate pieces on spaces, tabs, newlines and end-of-file.
-> More information: <https://pubs.opengroup.org/onlinepubs/9699919799/utilities/xargs.html>.
+> See also: `find`.
```

Comment:
```suggestion
```

Reply:
There is a reference to `find` at the bottom of two pages. Why do you suggest to remove `See also: find`>

Reply:
The functions of `xargs` and `find` are not related.